File: core/x/javaapi/codegen/src/test/resources/golden/expected/com/example/grts/OrderStatus.java
package com.example.grts;

import viaduct.java.api.reflect.Type;
import viaduct.java.api.types.GraphQLEnum;

public enum OrderStatus implements GraphQLEnum {
    PENDING,
    CONFIRMED,
    COMPLETED,
    CANCELLED;

    public static final Type<OrderStatus> Reflection = Type.ofClass(OrderStatus.class);
}

File: core/tenant/codegen/src/test/resources/golden/expected/grts/OrderStatus.kt
@file:Suppress("warnings")

package viaduct.api.grts

enum class OrderStatus : viaduct.api.types.Enum {
    PENDING,
    CONFIRMED,
    COMPLETED,
    CANCELLED;

    @OptIn(viaduct.apiannotations.InternalApi::class)
    object Reflection : viaduct.api.reflect.Type<viaduct.api.grts.OrderStatus> {
        override final val name = "OrderStatus"
        override final val kcls = viaduct.api.grts.OrderStatus::class
    }
}